app/src/main/java/com/main/mealplanner/dto/MealPlan.kt

```diff
@@ -5,9 +5,14 @@ import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 import java.util.*
 
-@Entity(tableName="mealplan")
-data class MealPlan(var CookSchedule: Date, var RecipeId : String, var OwnerEmail : String, @PrimaryKey @SerializedName("id") var MealPlanId:Int = 0) {
+@Entity(tableName = "mealplan")
+data class MealPlan(
+    var cookSchedule: Date,
+    var recipeID: String,
+    var ownerEmail: String,
+    @PrimaryKey @SerializedName("id") var mealPlanId: Int = 0
+) {
```

Comment:
formatted corrected and change to camel case as is Kotlin standard

Reply:
One line DTO is fine, though multi line may be more readable.

Naming conventions came up in a previous code review.